src/components/bottom_bar/bottom_bar.tsx

```diff
@@ -59,26 +59,27 @@ export const POSITIONS = ['static', 'fixed', 'sticky'] as const;
 export type _BottomBarPosition = typeof POSITIONS[number];
 
 type _BottomBarExclusivePositions = ExclusiveUnion<
+  { position?: 'static' | 'sticky' },
   {
     position?: 'fixed';
     /**
-     * Whether to wrap in an OuiPortal which appends the component to the body element.
+     * Whether to wrap in OuiPortal. Can be configured using "insert" prop.
      * Only works if `position` is `fixed`.
      */
     usePortal?: boolean;
     /**
-     * Whether the component should apply padding on the document body element to afford for its own displacement height.
-     * Only works if `usePortal` is true and `position` is `fixed`.
+     * Configuration for placing children in the DOM. By default, attaches children to the body element.
+     * Only works if `position` is `fixed` and `usePortal` is true.
      */
-    affordForDisplacement?: boolean;
-  },
-  {
+    insert?: OuiPortalInsert;
     /**
-     * How to position the bottom bar against its parent.
+     * Whether the component should apply padding on the document body element to afford for its own displacement height.
+     * Only works if `position` is `fixed` and `usePortal` is true.
      */
-    position: 'static' | 'sticky';
```

Comment:
Just so I understand this correct, options are:

```ts
{
  position: 'static' | 'sticky'
}
```
or
```ts
{
  position: 'fixed',
  usePortal: ...,
  insert: ...,
  affordForDisplacement: ...,
}
```

i.e. just adding `insert` to `position: fixed`? Are there any combinations that worked previously but don't with this change?

Reply:
Nice catch! I wrote this a little wrong, I've rewritten this interface and I'm afraid it's going to be a breaking change

Reply:
I think that given the same code before and after this change, if the functionality remains consistent, we shouldn't consider it breaking. I.e. if I don't specify `usePortal` before and after this change, it should look and function the same, and that should be fine.

Reply:
I see what you mean. The point is that we had `usePortal` and `affordForDisplacement` as defaults `true` and now they are `false`, this change leads to different behavior if these properties were not explicitly specified. This PR in itself is breaking because we allow BottomBar of any position type to be portable, which means that the behavior that was the default for fixed position should apply to other types, but in order to avoid not-expected behavior when using BottomBar, we need to remove some defaults and specify them ourselves if needed